srcdoc/configure/base_directives.mt

```diff
@@ -412,6 +412,25 @@ EOT
 By setting the value to <code>OFF</code> and by using the <code>%{error}x</code> specifier of the <a href="configure/access_log_directives.html">access-log</a> directive, it is possible to log request-level errors only to the access log.
 ? })
 
+<?
+$ctx->{directive}->(
+    name    => "h2olog",
+    levels  => [ qw(global) ],
+    desc    => q{Settings for h2olog(1).},
+)->(sub {
+?>
+<?= $ctx->{example}->('Configure a UNIX domain socket for h2olog', <<'EOT')
+h2olog:
+  path: /path/to/h2olog.sock # required
+  appdata: ON # optional
+  owner: h2o # optional
+  group: h2o # optional
+  permission: 666 # optional
+  sndbuf: 65536 # optional
+EOT
```

Comment:
I think the directive is now host-level and the Unix socket is specified differently?

Reply:
Done in 62827f3.